pkg/framework/test/control_plane.go

```diff
@@ -0,0 +1,41 @@
+package test
+
+// ControlPlane is a struct that knows how to start your test control plane.
+//
+// Right now, that means Etcd and your APIServer. This is likely to increase in future.
+type ControlPlane struct {
+	APIServer ControlPlaneProcess
+}
+
+// ControlPlaneProcess knows how to start and stop a ControlPlane process.
+// This interface is potentially going to be expanded to e.g. allow access to the processes StdOut/StdErr
+// and other internals.
+type ControlPlaneProcess interface {
```

Comment:
I don't know if the word "Process" is fair, since it's not technically a process, but a bunch of processes.